lib/mix/tasks/nerves_hub.user.ex

```diff
@@ -154,7 +154,11 @@ defmodule Mix.Tasks.NervesHub.User do
     password = Shell.password_get("Local user password:")
 
     with :ok <- File.mkdir_p(path),
-         {:ok, %{key: key_pem, cert: cert_pem}} <- User.auth(password),
+         {:ok, %{key: key_der, cert: cert_der}} <- User.auth(password),
+         {:Certificate, _, _, _} = cert <- X509.from_der(cert_der, :Certificate),
```

Comment:
`X509.from_der` is deprecated. You want `X509.Certificate.from_der`. However, `User.auth/1` should return the Erlang record rather than the PEM or DER-formatted key and everyone should convert it how they see fit. That will remove a bunch of unnecessary (and confusing) transformations here and in other places. I didn't get to that refactor the other day. Sorry, I didn't see that I broke the export code yesterday.

Reply:
Right now they are stored as `der` binaries. Should I update `User.auth/1` to return records and update it's use everywhere?

Reply:
Yes, that would be great! Thanks!